distributed_index_tests/lib/syncgateway.py

```diff
@@ -23,5 +18,9 @@ def info(self):
     def stop(self):
         orch.syncgatewayactions.stop(self.host_name)
 
-    def start(self):
-        orch.syncgatewayactions.start(self.host_name)
+    def start(self, config):
```

Comment:
It was intentional to have to explicitly provide the configuration. The hope was to prevent accidental state in our scripts